charts/stable/qbittorrent/values.yaml

```diff
@@ -2,6 +2,12 @@ image:
   repository: ghcr.io/onedr0p/qbittorrent
   pullPolicy: IfNotPresent
   tag: 4.6.0@sha256:cb6259df3be61a5108bc4f85b09a523fc4b250589ca9b315ef927cbdf779afb6
+qbitportforwardImage:
+  repository: mjmeli/qbittorrent-port-forward-gluetun-server
+  pullPolicy: IfNotPresent
+  tag: latest@sha256:67d0d21ed792cf80716d4211e7162b6d375af5c12f3cf096c9032ad705dddaa8
+qbitportforward:
+  enabled: false
```

Comment:
QBT_* fields have to be added under here too

Reply:
like this?

Reply:
yes, but we can make it easier!

remove *_addr from both here and questions
and on the env section set them like this

qbit addr
```
{{ printf "http://%v:%v" (include "tc.v1.common.lib.chart.names.fullname" $) .Values.service.main.ports.main.port }}
```

gluetun

```
{{ printf "http://%v-gluetun:8000" (include "tc.v1.common.lib.chart.names.fullname" $) }}
```

(if 8000 port is configurable, we should link to the variable)

Reply:
Should I also add 8000 as a port and set it to clusterip?
Is there a way to have it only get added if the sidecar is enabled?

Reply:
8000 is gluetun's api port? Is that configurable?
We should probably need service for it, you can disable it like you do for the workload in the common.yaml

Reply:
It is configurable with a gluetun ENV. The default is 8000. Its always running.

Reply:
> It is configurable with a gluetun ENV. The default is 8000. Its always running.

Might remember wrong but isn't default 8888?

Reply:
https://github.com/qdm12/gluetun/blob/674f0a32095c306edda07862b0015a7217c94bb8/Dockerfile#L185
Looks like its 8000